idaes/core/surrogate/tests/test_pysmo_surrogate.py

```diff
@@ -1373,9 +1373,13 @@ def test_populate_block_unisurrogate_poly(self, pysmo_surr1):
         assert blk.outputs["z1"].bounds == (None, None)
         assert isinstance(blk.pysmo_constraint, Constraint)
         assert len(blk.pysmo_constraint) == 1
-        assert str(blk.pysmo_constraint["z1"].body) == (
-            "outputs[z1] - (-75.26111111111476 - 8.815277777775934*inputs[x1] + 18.81527777777826*inputs[x2] - 2.2556956302821618e-13*(inputs[x2]*inputs[x1]))"
-        )
+        cstr = str(blk.pysmo_constraint["z1"].body).replace("outputs[z1]", "10")
+        cstr = cstr.replace("inputs[x1]", "1")
+        cstr = cstr.replace("inputs[x2]", "5")
+        assert eval(cstr) == pytest.approx(0, abs=1e-8)
+        # assert str(blk.pysmo_constraint["z1"].body) == (
```

Comment:
Is this commented code still necessary?

Reply:
No, I just forgot to delete the old code.  Will remove.